app/controllers/api/v1/search_keywords_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SearchKeywordsController < ApplicationController
+      before_action :authorize!
+
+      def index
+        pagination, keywords = search_keywords_form.search_keywords
```

Comment:
Filtering by `current_user` *must* be handled by Pundit policy, not by the form or the query object 🙈